common/js/auth-buttons.js

```diff
@@ -46,4 +50,55 @@
     registration.searchParams.set('returnToUrl', location)
     location.href = registration
   }
-})(solid)
+
+  async function getAccountInfo () {
+    const SOLID = $rdf.Namespace('http://www.w3.org/ns/solid/terms#')
+
+    // load data
+    const accountUrl = new URL('/account', location).href
+    const accountStore = $rdf.graph()
+    const accountFetcher = new $rdf.Fetcher(accountStore)
+    await accountFetcher.load(accountUrl)
+
+    // get DOM and data-dependencies
+    const quotaNode = accountStore.any($rdf.sym(accountUrl), SOLID('storageQuota'))
+    const usageNode = accountStore.any($rdf.sym(accountUrl), SOLID('storageUsage'))
+    const storagePanel = document.getElementById('StoragePanel')
+    const storageQuota = document.getElementById('StorageQuota')
+    const storageUsage = document.getElementById('StorageUsage')
+    const storageUsedBar = document.getElementById('StorageUsedBar')
+    if (!quotaNode || !usageNode || !storagePanel || !storageQuota || !storageUsage || !storageUsedBar) {
```

Comment:
I suggest moving this check _before_ the fetch call. This would avoid any unneeded calls for pages that don't show the quota.

Reply:
Fixed in b79bd563d3c582b6babb7018149a55e45e53ebb6